spec/features/visitors/email_confirmation_spec.rb

```diff
@@ -55,7 +55,12 @@
 
   context 'confirmed user is signed in and tries to confirm again' do
     it 'redirects the user to the profile' do
-      sign_up_and_2fa
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
```

Comment:
what do you think would be a a good name for this method? it is no longer "sign up and 2fa" it is "sign up and 2fa not with LOA3" but that feels like a mouthful...